wordpress.yaml

```diff
@@ -1,7 +1,136 @@
 ---
-- name: Deploy Wordpress
+- name: Setting up wordpress
   hosts: all
   tasks:
+    - name: Update repositories 
+      become: true
+      apt:
+        #upgrade: yes
+        update_cache: yes
+        cache_valid_time: 86400 #A day
+      
+    - name: Install php dependencies
+      become: true
+      apt:
+        name:
+          - php-cli
+          - php-fpm
+          - php-mysql
+          - php-json
+          - php-opcache
+          - php-mbstring
+          - php-xml
+          - php-gd
+          - php-curl
+        state: present
+     
+    - name: Install nginx
+      become: true
+      apt:
+        name: nginx 
+        state: present
 
-    - name: message to remove
-      command: echo 'Here you gonna put all tasks and roles that gonna be runing on the VM'
+    - name: Open firewall for nginx
+      become: true
+      ufw:
+        rule: allow
+        name: 'Nginx Full'
+    
+    - name: Copy NGINX default settings file
+      become: true
+      copy:
+        src: /vagrant/default
+        dest: /etc/nginx/sites-enabled
+        force: yes
+
+    - name: Install MySQL Server
+      become: true
+      apt:
+        name: 
+          - mysql-server
+          - python3-pymysql  
```

Comment:
why do you need `python3-pymysql`?

Reply:
In order to use the ansible sql commands `mysql_db` and `mysql_user` I had to install `python3-pymysql` (Suggested also in the docs)

Reply:
ok, make sence